src/sparse_variational.jl

```diff
@@ -1,16 +1,72 @@
+raw"""
+    Centered()
+
+Used in conjunction with `SparseVariationalApproximation`.
+States that the `q` field of [`SparseVariationalApproximation`](@ref) is to be interpreted
+directly as the approximate posterior over the pseudo-points.
+
+This is also known as the "unwhitened" parametrization [1].
+
+See also [`NonCentered`](@ref).
+
+[1] - https://en.wikipedia.org/wiki/Whitening_transformation
 """
-    SparseVariationalApproximation(fz::FiniteGP, q::AbstractMvNormal)
+struct Centered end
 
-Packages the prior over the pseudo-points, `fz`, and the approximate posterior at the
-pseudo-points, `q`, together into a single object.
+raw"""
+    NonCentered()
+
+Used in conjunction with `SparseVariationalApproximation`.
+States that the `q` field of [`SparseVariationalApproximation`](@ref) is to be interpreted
+as the approximate posterior over `cholesky(cov(u)).L \ (u - mean(u))`, where `u` are the
+pseudo-points.
+
+This is also known as the "whitened" parametrization [1].
+
+See also [`Centered`](@ref).
+
+[1] - https://en.wikipedia.org/wiki/Whitening_transformation
 """
-struct SparseVariationalApproximation{Tfz<:FiniteGP,Tq<:AbstractMvNormal}
+struct NonCentered end
+
+struct SparseVariationalApproximation{Parametrization,Tfz<:FiniteGP,Tq<:AbstractMvNormal}
     fz::Tfz
     q::Tq
 end
 
 raw"""
-    posterior(sva::SparseVariationalApproximation)
+    SparseVariationalApproximation(::Parametrization, fz::FiniteGP, q::AbstractMvNormal)
```

Comment:
We don't have an abstract type `Parametrization` (should we have one ?) so the typing is misleading here

Reply:
I could imagine we could also have the Opper 2009 parametrization as another subtype for example!

Reply:
> We don't have an abstract type Parametrization (should we have one ?) so the typing is misleading here

Ah, right. I don't know that we need a super-type for now (I'm not sure that it adds anything at this point, since we don't define any functionality which all parametrisations ought to implement), so I'll just make the fact that it's a completely free type parameter clear

Reply:
```suggestion
    SparseVariationalApproximation(
        ::Parametrization, fz::FiniteGP, q::AbstractMvNormal
    ) where {Parametrization}
```

Reply:
@theogf is this clearer?

Reply:
Maybe `Union{Centered,NonCentered}` (which would be exact) ?

Reply:
But would that not suggest that this isn't extensible? For example, I don't want to suggest that someone shouldn't add a new parametrisation.

Reply:
Ah yeah, that's true. Then your proposition sounds good